0001-kaa-protocol/README.md

```diff
@@ -99,6 +99,9 @@ Thus, for them the resource paths start with:
 
 The rest of the resource path is extension-specific and is described in separate documents that define KP protocol extensions.
 
+### MQTT QoS
+KP uses MQTT QoS level 0.
```

Comment:
I agree to ignore this topic and assume it is implicit if the MQTT QoS is **just** the transport QoS - not the extension services reliability, etc.